packages/accounts/src/light-account/utils.ts

```diff
@@ -15,114 +14,294 @@ import {
+> = keyof IAccountVersionRegistry[TLightAccountType];
+
+export type LightAccountVersionDef<
+  TLightAccountType extends LightAccountType = LightAccountType,
+  TLightAccountVersion extends LightAccountVersion<TLightAccountType> = LightAccountVersion<TLightAccountType>
+> = IAccountVersionRegistry[TLightAccountType][TLightAccountVersion];
+
+export interface IAccountVersionRegistry {
+  LightAccount: {
+    /** @deprecated This version does not support 1271 signature validation */
+    "v1.0.1": AccountVersionDef<"LightAccount", "v1.0.1", "0.6.0">;
+    /** @deprecated This version has a known issue with 1271 validation */
+    "v1.0.2": AccountVersionDef<"LightAccount", "v1.0.2", "0.6.0">;
+    "v1.1.0": AccountVersionDef<"LightAccount", "v1.1.0", "0.6.0">;
+    /** LightAccount v2 Changelog.
+     * It is recommended to use v2.0.0
+     * https://alchemotion.notion.site/External-Light-Account-v2-Changelog-725b306ab1e04eb0a3e596521dd8f0bd */
+    "v2.0.0": AccountVersionDef<"LightAccount", "v2.0.0", "0.7.0">;
+  };
+  MultiOwnerLightAccount: {
+    /** MultiOwnerLightAccount v2 Changelog
+     * It is recommended to use v2.0.0
+     * https://alchemotion.notion.site/External-Light-Account-v2-Changelog-725b306ab1e04eb0a3e596521dd8f0bd */
+    "v2.0.0": AccountVersionDef<"MultiOwnerLightAccount", "v2.0.0", "0.7.0">;
+  };
+}
+
+export const AccountVersionRegistry: IAccountVersionRegistry = {
+  LightAccount: {
+    "v1.0.1": {
+      type: "LightAccount",
+      version: "v1.0.1",
+      address: toRecord<Chain, "id", { factory: Address; impl: Address }>(
+        supportedChains,
+        "id",
+        () => ({
+          factory:
+            "0x000000893A26168158fbeaDD9335Be5bC96592E2".toLowerCase() as Address,
+          impl: "0xc1b2fc4197c9187853243e6e4eb5a4af8879a1c0".toLowerCase() as Address,
+        })
+      ),
+      entryPointVersion: "0.6.0",
+    },
+    "v1.0.2": {
+      type: "LightAccount",
+      version: "v1.0.2",
+      address: toRecord<Chain, "id", { factory: Address; impl: Address }>(
+        supportedChains,
+        "id",
+        () => ({
+          factory:
+            "0x00000055C0b4fA41dde26A74435ff03692292FBD".toLowerCase() as Address,
+          impl: "0x5467b1947F47d0646704EB801E075e72aeAe8113".toLowerCase() as Address,
+        })
+      ),
+      entryPointVersion: "0.6.0",
+    },
+    "v1.1.0": {
+      type: "LightAccount",
+      version: "v1.1.0",
+      address: toRecord<Chain, "id", { factory: Address; impl: Address }>(
+        supportedChains,
+        "id",
+        () => ({
+          factory:
+            "0x00004EC70002a32400f8ae005A26081065620D20".toLowerCase() as Address,
+          impl: "0xae8c656ad28F2B59a196AB61815C16A0AE1c3cba".toLowerCase() as Address,
+        })
+      ),
+      entryPointVersion: "0.6.0",
+    },
+    "v2.0.0": {
+      type: "LightAccount",
+      version: "v2.0.0",
+      address: toRecord<Chain, "id", { factory: Address; impl: Address }>(
+        supportedChains,
```

Comment:
we should limit to this just the chain it's currently deployed on

Reply:
done. only on arb sepolia